src/control/cmd/daos/acl.go

```diff
@@ -413,11 +414,20 @@ func (cmd *containerSetOwnerCmd) Execute(args []string) error {
 
 	lType := C.get_dpe_val(&entries[0])
 	if lType == C.DAOS_PROP_CO_LAYOUT_POSIX {
+		if cmd.NoCheck {
+			return errors.New("--no-check is not compatible with POSIX containers")
+		}
+
```

Comment:
i believe your suggestion originally was to actually support the chown but in that case we do it only for the container and not for the root object that requires the uid/gid.
I believe another option that was suggested by Jeff would be to pass the uid and gid manually in addition to the user and group.

Reply:
I think Jeff's suggestion is the least surprising of the two. I'll add that. Thanks for the reminder.